src/index.ts

```diff
@@ -0,0 +1,77 @@
+import { PinejsClientCoreFactory } from 'pinejs-client-core';
+export type { PinejsClientCoreFactory } from 'pinejs-client-core';
+
+interface BackendParams {
+	/** The browser fetch API implementation or a compatible one */
+	fetch?: typeof fetch;
+}
+
+type PromiseObj = Promise<{}>;
+
+class RequestError extends Error {
```

Comment:
It'd be nice to export this so downstream can do `err instanceof RequestError` as necessary to check for the specific error class rather than trying to infer it from the properties - especially since I don't think the typing will be exported currently